loadtests/README.md

```diff
@@ -9,12 +9,48 @@ For now using the following version of JMeter:
 `https://dlcdn.apache.org/jmeter/binaries/apache-jmeter-5.5.tgz`
 
 
-# Some scripts (WIP)
+```
+test-data
+├── files-dyn                    # artipie files adapter dynamic data test
+│   ├── files-list.csv           # list of file names for download testing
+│   ├── download-files-csv.jmx
+│   └── upload-files.jmx
+├── maven-dyn                    # artipie maven adapter dynamic data test
+│   ├── files-list.csv
+│   └── repository               # generated by prep-maven-dyn.py
+└── maven-real                   # artipie maven adapter real-artifacts data test
+    ├── files-list.csv
+    └── repository               # generated by prep-maven-real.sh
+```
+
+## Test scripts
 
  - `jmeterReport.sh` allows to generate report files for given JMeter jmx file.
  - `Dockerfile.client` allows to run for recording `conan install` command with given Conan server URL and JMeter proxy URL
  - `prep-maven.sh` downloads required maven artifact files for tests. Results put in `upload-maven-src/repository/`
 
-# Downloading via proxy (JMeter)
-time mvn clean install -Dhttp.proxyHost=localhost -Dhttp.proxyPort=8888 -Dhttps.proxyHost=localhost -Dhttps.proxyPort=8888 -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.skip -Ddockerfile.skip=true
+## Maven test data generator parameters
+
+Maven test data generator `prep-maven-dyn.py` requires all generator parameters provided for the command line. Here's the command format:
+```
+./gen-maven.py groups artifacts versions big_size med_size small_size bigs_percent smalls_percent
+```
+For example:
+```
+./prep-maven-dyn.py 4 5 6 9000000 192000 16386 5 30
+```
+Paramets description:
+- `groups` number of maven groups to generate 
+- `artifacts` number of artifacts inside the group to generate
+- `versions` number of versions per artifactId
+- `big_size` size of "big" artifact type in bytes
+- `med_size` size of "medium" artifact type in bytes
+- `small_size` size of "small" artifact type in bytes
+- `bigs_percent` percent of "big" artifacts (jars) to generate
+- `smalls_percent` percent of "small" artifacts. The rest will be "medium" sized.
```

Comment:
`mediums_percen` looks more logical for me

Reply:
@dgarus , ok, I'll switch to specifying bigs & mediums, so small ones will be calculated.